jina/peapods/pods/k8slib/kubernetes_deployment.py

```diff
@@ -101,18 +105,32 @@ def deploy_service(
         custom_resource_dir=custom_resource_dir,
     )
 
+    kubernetes_tools.create(
```

Comment:
I think it is important to create the config map before you create the deployment.